CHANGELOG.md

```diff
@@ -13,6 +13,65 @@ All notable changes to this project will be documented in this file.
 This project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.37.0]
+
+### News around this release
+
+* [Jake Rembish](https://github.com/rembishj) has been undergoing his PhD with
+  the output of his work being contributed to PyMedPhys. This release coincides
+  with the culmination of Jake's PhD and this version will be the one
+  referenced within his dissertation. It has been brilliant to see both Jake
+  and his project to grow be what it is today. Thank you Jake 😊.
```

Comment:
and his project to grow be what it is today

->

and his project to grow to be what it is today